lemminx-liberty/src/main/java/io/openliberty/lemminx/liberty/services/FeatureService.java

```diff
@@ -59,13 +70,14 @@ private FeatureService() {
     // Only need the public features
     ArrayList<Feature> publicFeatures = readPublicFeatures(reader);
 
+    LOGGER.info("returning public features: " + publicFeatures.size());
```

Comment:
Maybe make this more descriptive? something like
"Returning public features from maven"